packages/MarkdownEditor-Core.package/MarkdownEmphasisDelimiter.class/instance/attributeStartingAt..st

```diff
@@ -1,9 +1,9 @@
 attributes
-attributeStartingAt: openingMarkdownEmphasisDelimiter
+attributeStartingAt: openingEmphasisDelimiter
 
-	| attributeDelimiterLength start end attributeClass |
-	attributeDelimiterLength := (self delimiterLengthForAttributeWith: openingMarkdownEmphasisDelimiter).
-	start := openingMarkdownEmphasisDelimiter endPosition - attributeDelimiterLength + 1.
-	end := self startPosition + attributeDelimiterLength - 1.
-	attributeClass := self attributeTypeFor: attributeDelimiterLength.
-	^ attributeClass from: start to: end
+	| delimiterLength start end |
+	delimiterLength := self delimiterLengthForAttributeWith: openingEmphasisDelimiter.
+	start := openingEmphasisDelimiter endPosition - delimiterLength + 1.
```

Comment:
I'd rename these inline variables if the line does not get too long

Reply:
To what? The line lengths would be okay, I guess.

Reply:
I wanted to rename them `startPosition` or `endPosition`, but after the discussion about the arguments `attributeFrom:to:`, I'd keep them here and rename the arguments of `attributeFrom: start to: end`